sympy/ntheory/factor_.py

```diff
@@ -2201,3 +2201,29 @@ def is_deficient(n):
     if is_perfect(n):
         return False
     return bool(abundance(n) < 0)
+
+
+def is_amicable(n, N):
+    """Returns True if the pair ``(n, N)`` is a Amicable Number, else False.
+
+    Amicable numbers are two different numbers so related that the sum of the proper divisors of each is equal to the other number.
+
+    Examples
+    ========
+
+    >>> from sympy import *
```

Comment:
the previous import was ok, importing `*` will be a bit costly

Reply:
@Yathartha22 but it does not pass the travis CI, locally it runs well, Any advise would be greatly appreciated